crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -125,6 +125,15 @@ def repocheck
       end
     end
 
+    def ha_deployed?
+      founders = NodeObject.find("pacemaker_founder:true AND pacemaker_config_environment:*")
+
+      if founders.empty?
+        Rails.logger.warn("No pacemaker cluster is configured.")
```

Comment:
I would just return `founders.any?`, as I think the log entry is not really necessary